public/apps/configuration/panels/internal-user-edit/internal-user-edit.tsx

```diff
@@ -59,6 +60,8 @@ function createErrorToast(id: string, failedAction: string): Toast {
 
 export function InternalUserEdit(props: InternalUserEditDeps) {
   const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const [isPasswordInvalid, setIsPasswordInvalid] = useState(false);
```

Comment:
Learnt from Mihir on one of my PRs that it is better to add type to useState<boolean>

Reply:
Sure, will add.